src/main/java/com/mathworks/ci/MatlabBuilder.java

```diff
@@ -148,8 +149,9 @@ public DescriptorExtensionList<TestRunTypeList, Descriptor<TestRunTypeList>> get
          * descriptor class.
          */
 
-
+        @POST
         public FormValidation doCheckMatlabRoot(@QueryParameter String matlabRoot) {
+            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
```

Comment:
This breaks the UI for users with Job/Configure permission:
<img width="1201" alt="image" src="https://github.com/mathworks/jenkins-matlab-plugin/assets/91559310/481a7bff-cecb-49e4-923b-b77e6c4f6ba1">

Reply:
I am using Permission.CREATE now would that be the safe one to check ?

Reply:
See https://www.jenkins.io/doc/developer/security/form-validation/#checking-permissions
I see that you're not using CREATE, but CONFIGURE. It has to be checked on the correct context. Please see the docs above, second code example there. `@AncestorInPath` is what you're interested in.

Reply:
Done.